.github/workflows/update.yml

```
@@ -0,0 +1,75 @@
+name: Update Components
+
+on:
+  schedule:
+    - cron: "0 * * * *"
```

(Without knowing what the precise runtime of the action is)

```suggestion
    - cron: "*/10 * * * *"
```